### PR TITLE
PROD jobs and pipelines DSLs

### PR DESCRIPTION
**Pull request by almope**
DSLs for the following jobs in the PROD folder:
- rhba-properties-generator
- offline-repo-builder
- rhba-staging

### CODE REVIEW

**Review by mbiarnes — APPROVED** [review]
LGTM

File: job-dsls/jobs/prod_offline_repo_builder.groovy [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    description("Check offliner manifest against offline repository")
+
+    logRotator {
+        numToKeep(5)
+        artifactNumToKeep(1)
+    }
+
+    parameters {
+        stringParam("MANIFEST_URL", "", "Offliner manifest URL")
+        stringParam("RELEASE_REPO_URL", "http://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8081/nexus/content/repositories/offline-repo-7.6/", "Scratch repository for the release")
```
Comment (mareknovotny) [inline job-dsls/jobs/prod_offline_repo_builder.groovy:18]: is that URL of repo correct @winklerm ? i am confused by the number 7.6 only ;)

Comment (winklerm) [reply job-dsls/jobs/prod_offline_repo_builder.groovy:18]: Good point - it is outdated; on the other hand, this value is going to be changed for each release, so this particular URL is just a default that needs to be overwritten when running the job.

Anyway, I guess we could put a recent value there, or - maybe better - something like:

```suggestion
        stringParam("RELEASE_REPO_URL", "http://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8081/nexus/content/repositories/offline-repo-REPLACE-ME-WITH-MAJOR.MINOR/", "Scratch repository for the release")
```

Wdyt?

**Review by winklerm — APPROVED** [review]
Looks good to me, just one potential improvement (not a blocker from my POV).

File: job-dsls/jobs/prod_offline_repo_builder.groovy (hunk 1 above)
Comment (almope) [reply job-dsls/jobs/prod_offline_repo_builder.groovy]: I went with the update of the parameters to recent values as many times they serve as both a default value and an example. I think it's simpler to understand than using place holders or anything similar.

File: job-dsls/jobs/seed_job.groovy [hunk 2]
```diff
@@ -77,7 +77,10 @@ job("a-seed-job") {
                     "job-dsls/jobs/**/reduced_drools_release.groovy \n" +
                     "job-dsls/jobs/**/prod_projects_downstream_production.groovy \n" +
                     "job-dsls/jobs/**/srcclr_scan_job.groovy\n" +
-                    "job-dsls/jobs/**/jenkins_shared_libs.groovy")
+                    "job-dsls/jobs/**/jenkins_shared_libs.groovy\n" +
+                    "job-dsls/jobs/**/prod_offline_repo_builder.groovy\n" +
```
Comment (almope) [inline job-dsls/jobs/seed_job.groovy:81]: @mareknovotny I've added the jobs to the main seed job. Thanks!

**Pull request merged**